### PR TITLE
[ACA-4487] support search forms layout

### PR DESCRIPTION
**Pull request by dhrn**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**
expansion layout replaced with chip based search layout 

image
image

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

depends https://github.com/Alfresco/alfresco-ng2-components/pull/7122

### CODE REVIEW

File: e2e/resources/extensibility-configs/context-submenus-ext.json [hunk 1]
```diff
@@ -262,18 +262,6 @@
       }
     ],
     "toolbar": [
-      {
-        "id": "app.toolbar.searchFilter",
-        "order": 50,
-        "title": "APP.BROWSE.SEARCH.TOGGLE_SEARCH_FILTER",
```
Comment (DenysVuika) [inline e2e/resources/extensibility-configs/context-submenus-ext.json:268]: Please check if this i18n key is not used anywhere else and remove from translations

File: e2e/resources/extensibility-configs/context-submenus-ext.json [hunk 2]
```diff
@@ -262,18 +262,6 @@
...
-        "icon": "view_list",
-        "actions": {
-          "click": "TOGGLE_SEARCH_FILTER"
-        },
-        "rules": {
-          "visible": "app.navigation.isSearchResults"
```
Comment (DenysVuika) [inline e2e/resources/extensibility-configs/context-submenus-ext.json:274]: check if it's not used anywhere and remove the rule as well

**Review by DenysVuika — CHANGES_REQUESTED** [review]
looks good but please check if we can cleanup some i18n/rule resources as well

**Review by eromano — APPROVED** [review]
LGTM after fixed the Denys comment

File: e2e/suites/actions-available/files-folders/generic.test.ts [hunk 3]
```diff
@@ -150,13 +149,6 @@ describe('Generic tests : ', () => {
       await page.goToFavoriteLibraries();
       expect(await toolbar.isEmpty()).toBe(true, `actions displayed though nothing selected`);
     });
-
-    it('[C291815] on Search Results', async () => {
```
Comment (adinapitul) [inline e2e/suites/actions-available/files-folders/generic.test.ts:154]: Why are you removing this test?

File: e2e/suites/search/search-filters.test.ts [hunk 4]
```diff
@@ -285,52 +288,58 @@ describe('Search filters', () => {
...
-      expect(await page.getResultsChipsValues()).toEqual(['Adobe PDF Document']);
 
+      await fileTypeFilter.openDialog();
       await fileTypeFilter.checkCategory('JPEG Image');
+      await fileTypeFilter.clickApplyButton();
 
       expect(await dataTable.isItemPresent(filePdfUser2.name)).toBe(true, 'PDF file not displayed');
       expect(await dataTable.isItemPresent(fileJpgUser1.name)).toBe(true, 'JPG file not displayed');
-      expect(await page.getResultsChipsValues()).toEqual(['Adobe PDF Document', 'JPEG Image']);
+      await expect(await fileTypeFilter.getChipTitle()).toEqual(['Adobe PDF Document', 'JPEG Image'].join(', '));
```
Comment (adinapitul) [inline e2e/suites/search/search-filters.test.ts:321]: await before expect is not needed in ACA. Please remove for consistency

File: e2e/suites/actions-available/files-folders/generic.test.ts (hunk 3 above)
Comment (dhrn) [reply e2e/suites/actions-available/files-folders/generic.test.ts:154]: This test is not valid anymore. (toggle search filter action has been removed)

Comment (adinapitul) [reply e2e/suites/actions-available/files-folders/generic.test.ts:154]: The test is still valid, check the name of the describe.
Just that now you need another way of checking this.
Might work with toolbar.isEmpty() to be true

**Pull request merged**